readme.md

```diff
@@ -1,39 +1,45 @@
-OWASP Nettacker
-=========
+<h1 align="center">
```

Comment:
Thank you for the PR.
it's markdown and not sure why have you included html in that. can you explain?

Reply:
Thanks for reviewing it , it is for better _formatting and indentations_, please see it in preview mode to see the outputs in specific manner.